cli/common.go

```diff
@@ -36,10 +36,21 @@ func fillCommonRuntime() (*torcx.CommonConfig, error) {
 		ConfDir: torcx.DefaultConfDir,
 		StorePaths: []string{
 			torcx.VendorStoreDir,
-			torcx.OemStoreDir,
 		},
 	}
 
+	// Determine OS release ID
+	OsRelease, err := torcx.CurrentOsVersionID(torcx.OsReleasePath)
+	if err != nil {
```

Comment:
Yep, better. Debug? Warn? Info? This is in the path of basically every command, so it may be a become annoying and outside of user-control.